website/docgen/dgeni-config.js

```diff
@@ -57,29 +60,31 @@ myPackage.factory(require('./inline_tags/code'))
  *     and webdriver.js.
  * add-toc: Generates the table of contents.
  */
-myPackage.processor(require('./processors/tag-fixer'));
-myPackage.processor(require('./processors/these-children'));
-myPackage.processor(require('./processors/filter-jsdoc'));
-myPackage.processor(require('./processors/set-file-name'));
-myPackage.processor(require('./processors/transfer-see'));
-myPackage.processor(require('./processors/add-links'));
-myPackage.processor(require('./processors/filter-promise'));
-myPackage.processor(require('./processors/add-toc'));
+protractorPackage.processor(require('./processors/tag-fixer'));
+protractorPackage.processor(require('./processors/these-children'));
+protractorPackage.processor(require('./processors/filter-jsdoc'));
+protractorPackage.processor(require('./processors/set-file-name'));
+protractorPackage.processor(require('./processors/transfer-see'));
+protractorPackage.processor(require('./processors/add-links'));
+protractorPackage.processor(require('./processors/filter-promise'));
+protractorPackage.processor(require('./processors/add-toc'));
 
-myPackage.config(function(readFilesProcessor, templateFinder, writeFilesProcessor) {
+protractorPackage.config(function(readFilesProcessor, readTypeScriptModules, templateFinder, writeFilesProcessor) {
+  readTypeScriptModules.sourceFiles = [{
+    include: 'lib/**/*.ts',
+    basePath: 'lib'
+  }];
 
-  // Go to the protractor project root.
-  readFilesProcessor.basePath = path.resolve(__dirname, '../..');
+  readTypeScriptModules.hidePrivateMembers = true;
 
-  readFilesProcessor.sourceFiles = [
-    {include: 'built/browser.js'},
-    {include: 'built/element.js'},
-    {include: 'built/locators.js'},
-    {include: 'built/expectedConditions.js'},
-    {include: 'lib/selenium-webdriver/locators.js'},
-    {include: 'lib/selenium-webdriver/webdriver.js'},
-    {include: 'lib/webdriver-js-extender/index.js'}
-  ];
+  // Go to the protractor project root.
+  // --- JS
+  readFilesProcessor.basePath = path.resolve(__dirname, '..');
+  readFilesProcessor.sourceFiles = [{
+      include: 'lib/**/*.js',
```

Comment:
I think this `*.js` is probably dangerous - we only want to get the specific files we listed before. I guess just the `selenium-webdriver` ones in this case.